docs/diagnostic-logs.md

```diff
@@ -0,0 +1,164 @@
+
+<a href="properties-table"></a>
+**Properties Table**
+
+Name | Description
+------- | -------
+type | Collection of the log event. Currently, we provide information about connectivity to the Azure SignalR Service. Only `ConnectivityLogs` collection is available
+collection | Type of the log event. Allowed values are: `Connection`, `Authorization` and `Throttling`
+connectionId | Identity of the connection
+transportType | Transport type of the connection. Allowed values are: `Websockets` \| `ServerSentEvents` \| `LongPolling`
+connectionType | Type of the connection. Allowed values are: `Server` \| `Client`. `Server`: connection from server side; `Client`: connection from client side
+userId | Identity of the user
+message | Detailed message of log event
+
+The following code is an example of an archive log JSON string:
+
+```json
+{
+    "properties": {
+        "message": "Entered Serverless mode.",
+        "type": "ConnectivityLogs",
+        "collection": "Connection",
+        "connectionId": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
+        "userId": "User",
+        "transportType": "WebSockets",
+        "connectionType": "Client"
+    },
+    "operationName": "ServerlessModeEntered",
+    "category": "AllLogs",
+    "level": "Informational",
+    "callerIpAddress": "xxx.xxx.xxx.xxx",
+    "time": "2019-01-01T00:00:00Z",
+    "resourceId": "/SUBSCRIPTIONS/XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX/RESOURCEGROUPS/XXXX/PROVIDERS/MICROSOFT.SIGNALRSERVICE/SIGNALR/XXX",
+    "location": "xxxx"
+}
+```
+
+### Archive logs schema for Log Analytics
+
+To view diagnostic logs, follow these steps:
+
+1. Click `Logs` in your target log analytics.
+
+    ![Log Analytics menu item](./images/diagnostic-logs/log-analytics-menu-item.png)
+
+1. Enter `SignalRServiceDiagnosticLogs` and select time range to query diagnostic logs. For advanced query, please see [Get started with Log Analytics in Azure Monitor](https://docs.microsoft.com/en-us/azure/azure-monitor/log-query/get-started-portal)
+
+    ![Query log in Log Analytics](./images/diagnostic-logs/query-log-in-log-analytics.png)
+ 
+Archive log columns include elements listed in the following table:
+
+Name | Description
+------- | ------- 
+TimeGenerated | Log event time
+Collection | Type of the log event. Allowed values are: `Connection`, `Authorization` and `Throttling`
+OperationName | Operation name of the event
+Location | Location of your Azure SignalR Service
+Level | Log event level
+CallerIpAddress | IP address of your server/client
+Message | Detailed message of log event
+UserId | Identity of the user
+ConnectionId | Identity of the connection
+ConnectionType | Type of the connection. Allowed values are: `Server` \| `Client`. `Server`: connection from server side; `Client`: connection from client side
+TransportType | Transport type of the connection. Allowed values are: `Websockets` \| `ServerSentEvents` \| `LongPolling`
+
+## Trouble Shooting With Diagnostic Logs
```

Comment:
Is there any troubleshooting guide for `Authorization` and `Throttling` category?

Reply:
updated